packages/react-scripts/template/public/index.html

```diff
@@ -5,12 +5,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <!--
-      Notice the use of %PUBLIC_URL% in the tag above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+      Notice the use of the PUBLIC_URL variable in the tag above.  It will
+      be (or has been) replaced with the URL of the `public` folder during
+      the build.  Only files inside the `public` folder can be referenced
+      from the HTML.
 
-      Unlike "/favicon.ico" or "favico.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
+      Unlike "/favicon.ico" or "favicon.ico", the link href path will, if
```

Comment:
To be honest I don't think this is much clearer. Maybe we could provide some example?